Makefile

```diff
@@ -119,10 +119,10 @@ rundev:
 run:
 ifeq ("$(MF_BROKER_TYPE)", "rabbitmq")
 	sed -i "s,file: brokers/.*.yml,file: brokers/rabbitmq.yml," docker/docker-compose.yml
-	sed -i "s,MF_BROKER_URL: .*,MF_BROKER_URL: $$\{MF_RABBITMQ_URL\}," docker/docker-compose.yml
```

Comment:
I would probably consider just having `MF_BROKER_URL` and then let user configure it in different ways, depending on a broker type.

I.e. I am not sure if these fine-grained `MF_RABBITMQ_USER` and similar ENV vars are necessary for Mainflux, they look like overkill to me.

Reply:
@0x6f736f646f would it be possible here to use `MF_BROKER_TYPE` to construct `MF_RABBITMQ_URL` - something like:
```bash
MF_${MF_BROKER_TYPE^^}_URL
```
i.e. simply by translating `$MF_BROKER_TYPE` to uppercase and concatenating?

If yes, maybe we can avoid/simplify  if-else.

This is not necessary, I am just curious if it can be more elegant (but maybe it will be less readable).